shared/common-adapters/markdown.desktop.js

```diff
@@ -192,6 +193,8 @@ function messageCreateComponent(type, key, children, options) {
 
 const linkRegex = /^( *)((https?:\/\/)?[\w-]+(?<tld>\.[\w-]+)+\.?(:\d+)?(\/\S*)?)\b/i
 const inlineLinkMatch = SimpleMarkdown.inlineRegex(linkRegex)
+const mentionMatch = SimpleMarkdown.inlineRegex(/^\B@((?:[a-z]|[A-Z]|[0-9])(?:[a-z]|[A-Z]|[0-9]|_){1,15})\b/)
```

Comment:
Why have this generic regex instead of making a specific one with the mentionsAt prop?